test/check-session

```diff
@@ -39,10 +39,8 @@ class TestSession(MachineCase):
                 # HACK - sometimes the session gets stuck forever in
                 #        state 'closing' with no processes.
                 #        https://bugs.freedesktop.org/show_bug.cgi?id=89024
-                try:
-                    wait(cond)
-                except:
-                    raise RepeatThis("https://bugs.freedesktop.org/show_bug.cgi?id=89024")
+                # wait(cond)
```

Comment:
The `wait(cond)` is important.  Without RepeatThis, we don't need the `if` at all, we can just do `wait(cond)` always.